scheduler/proxy_test.go

```diff
@@ -83,7 +83,7 @@ func TestFifoChanges(t *testing.T) {
 		t.Helper()
 
 		client := net.NewClient(net.Options{
-			ResponseHeaderTimeout: 2 * time.Second,
+			ResponseHeaderTimeout: 3 * time.Second,
```

Comment:
why was this changed?

Reply:
It was flaky and was failing most of the time in my local environment, increasing the timeout solved the problem.